BackofficeBundle/Resources/public/ecmascript/Lib/jquery.initialize.js

```diff
@@ -50,4 +73,9 @@
     $.fn.initialize = function (callback) {
         msobservers.initialize(this.selector, callback);
```

Comment:
this.selector is deprecated (https://api.jquery.com/selector/). To fix this comment and the previous one, on selector always need to be an id, you could maybe use https://api.jqueryui.com/uniqueId/